cypress/integration/treasuries_spec.ts

```diff
@@ -86,10 +87,36 @@ describe('Create a new treasury', () => {
     cy.contains(editedDesc).should('be.visible')
   })
 
+  it('Should export user data', () => {
+    cy.visit('http://localhost:3000/config')
+    cy.contains('Export User Data').click()
+    cy.wait(2000)
+    const downloadsFolder = Cypress.config('downloadsFolder')
+    const downloadedFilename = path.join(downloadsFolder, 'roundtable-backup.mainnet.json')
+    cy.readFile(downloadedFilename).then(data => {
+      expect(data['treasuries'][0]['name']).to.equal('Test treasuryadded')
+      expect(data['treasuries'][0]['description']).to.equal('This is a description of the treasuryxxx')
+      expect(data['treasuries'][0]['script']).to.equal('gwMCgoIAWBz2v/tCURgnKF2D4mFnXqjUTGnZS54dlZQ1d90KggBYHOHZFcEMhAAXvTkIioJQeycVCkOOiQd4QiFJEwk=')
+    })
+  })
+
   it('Should delete treasury info', () => {
+    cy.visit('http://localhost:3000/treasuries/gwMCgoIAWBz2v%2FtCURgnKF2D4mFnXqjUTGnZS54dlZQ1d90KggBYHOHZFcEMhAAXvTkIioJQeycVCkOOiQd4QiFJEwk%3D')
     cy.contains('Edit Info').click()
     cy.contains('Delete').click()
     cy.url().should('eq', 'http://localhost:3000/treasuries/new')
     cy.contains(treasuryName + 'added').should('not.exist')
   })
-})
+
+  it('Should import user data', () => {
+    cy.visit('http://localhost:3000/config')
+    const downloadsFolder = Cypress.config('downloadsFolder')
+    const downloadedFilename = path.join(downloadsFolder, 'roundtable-backup.mainnet.json')
```

Comment:
I think you could use `[downloadsFolder, 'roundtable-backup.mainnet.json'].join('/')` here. Path package might not be needed.